src/sage/graphs/matching_covered_graph.py

```diff
@@ -2003,6 +2004,349 @@ def has_perfect_matching(G, algorithm='Edmonds', solver=None, verbose=0,
         raise ValueError('algorithm must be set to \'Edmonds\', '
                          '\'LP_matching\' or \'LP\'')
 
+    @doc_index('Bricks, braces and tight cut decomposition')
+    def is_brace(self, coNP_certificate=False):
+        r"""
+        Check if the (matching covered) graph is a brace.
+
+        A matching covered graph which is free of nontrivial tight cuts is
+        called a *brace* if it is bipartite. Let `G := (A \cup B, E)` be a
+        bipartite matching covered graph on six or more vertices. The
+        following statements are equivalent [LM2024]_:
+
+        1. `G` is a brace (aka free of nontrivial tight cuts).
+        2. `G - a_1 - a_2 - b_1 - b_2` has a perfect matching for any two
+           distinct vertices `a_1` and `a_2` in `A` and any two distinct
+           vertices `b_1` and `b_2` in `B`.
+        3. `G` is two extendable (any two nonadjacent distinct edges can be
+           extended to some perfect matching of `G`).
+        4. `|N(X)| \geq |X| + 2`, for all `X ⊂ A` such that `0 < |X| <
+           |A| - 1`, where `N(S) := \{b | (a, b) \in E \^ a \in S\}` is called
```

Comment:
```suggestion
           |A| - 1`, where `N(S) := \{b | (a, b) \in E ∧ a \in S\}` is called
```

though if Unicode characters is not encouraged then use `\wedge` instead. (plus LaTeX (compile to PDF) might not take Unicode characters)